.github/workflows/dbt-scheduled.yml

```
@@ -67,18 +72,27 @@ on:
           - weekly
           - monthly-full-refresh
+          - dev-monthly-full-refresh
           - snapshots
           - sdl-intraday
 
+# Dev refreshes also share the merge-validation lock before joining the prod queue.
+# Other schedules retain independent workflow runs and use only the job queue below.
+concurrency:
+  group: ${{ (inputs.run_type == 'dev-monthly-full-refresh' || github.event.schedule == '17 19 1 * *') && 'dbt-pr-validation-merge-queue' || format('dbt-scheduled-{0}', github.run_id) }}
+  cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)